packer/ubuntu-14.04_amd64-amis.json

```diff
@@ -3,7 +3,6 @@
     "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
     "aws_secret_key": "{{env `AWS_ACCESS_KEY`}}",
     "mesos_version": "0.22.1-1.0.ubuntu1404",
-    "marathon_version": "0.8.2-1.0.306.ubuntu1404",
```

Comment:
We'll still need marathon version but it would be the version of the image which is hardcode in install_docker.sh at the moment.